include/swift/SILOptimizer/Differentiation/PullbackEmitter.h

```diff
@@ -307,6 +307,13 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
     return pullbackTrampolineBBMap.lookup({originalBlock, successorBlock});
   }
 
+  //--------------------------------------------------------------------------//
+  // Debugging utilities
+  //--------------------------------------------------------------------------//
+
+  void debugAdjointValueMapping();
```

Comment:
You might want to mention “print” in the name here. “Debug” is vague.

Reply:
Oops, I totally meant "dump" instead of "debug". Changed to "print".